source/mir/math/func/normal.d

```diff
@@ -82,19 +100,76 @@ T normalDistribution(T)(const T a)
 }
 
 /// ditto
+deprecated("normalDistribution renamed, use normalCDF instead")
 T normalDistribution(T)(const T x, const T mean, const T stdDev)
     if (isFloatingPoint!T)
 {
     return normalDistribution((x - mean) / stdDev);
 }
 
-version(mir_test)
+version(mir_test_deprecated)
```

Comment:
We don't need to test deprecated API. Test should test only new names. Aliases ensures code is valid.

Reply:
Done